lib/LoginStrategy.ts

```diff
@@ -75,6 +82,13 @@ export class LoginStrategy extends BaseStrategy {
       acceptedRoles,
       ...options
     }: LoginStrategyOptions,
+    private issuerResolver: IssuerResolver,
+    private revokerResolver: RevokerResolver,
+    private credentialResolver: CredentialResolver,
+    private verifyProof: (
+      vc: string,
+      proof_options: string
+    ) => Promise<unknown>,
```

Comment:
`Unknown` is replacement for `any`, since `eslint` does not allow `any` and gives a warning.

Reply:
Right, doesn't `verifyProof` need to return a result though? If so, can we describe the type of that result?

Or is the expectation that it throws an exception if verification fails? If that's the case, I would expect the return type to be `Promise<void>`.

Perhaps we could add constructor documentation to clarify this?

Reply:
It would return a result, but then that result would be based on the implementation of this callBack method. So for ex, `verifyCredential` from `didkit-wasm-node` would return differently then some other method from other ssi-library and the same behaviour would be for throwing errors. This is why it is unknown. 

I would add some documentation.

Reply:
Hmm, I don't think the interface is very valuable if it can't be trusted at all. I think we should require that implementations conform to an interface. So if one wants to use `didkit-wasm-node` or a different implementation, one should need to write an adapter around it so that it conforms to the interface, IMO